.github/workflows/deduplicate-yarn.yml

```diff
@@ -0,0 +1,43 @@
+name: ⚙️ Deduplicate yarn.lock
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
I'll add `dev` & rebase onto `dev` as well 👍

Reply:
Sorry I meant `dev` instead of `main`, not in addition to.  That way any change to `yarn.lock` goes through our release cycle and doesn't go straight into `main` to be extra safe

```suggestion
```